pkg/scanner/local/scan.go

```diff
@@ -424,8 +425,14 @@ func excludeDevDeps(apps []ftypes.Application, include bool) {
 	if include {
 		return
 	}
+	var once sync.Once
```

Comment:
nit: It's completely the same, and you don't have to change. I just want to try something new 😄 OnceFunc and OnceValue were added in Go 1.21.
https://pkg.go.dev/sync#OnceFunc

```suggestion
	onceInfo := sync.OnceFunc(func(){
		log.Logger.Info("Suppressing dependencies for development and testing. To display them, try the '--include-dev-deps' flag.")
	})
```

Reply:
Why not apply your changes.
They look neater.
Thanks for this information!

Changed in https://github.com/aquasecurity/trivy/pull/6211/commits/03249ca551e2c030be36fd1669bbb94017f96430